### DISCUSSION TITLE
I can't run the second command.

### DISCUSSION THREAD

**Discussion by naza1144** [Rapptz/discord.py#7763] [Q&A] [answered]
![Screenshot 2022-03-26 194409](https://user-images.githubusercontent.com/85209746/160240043-42873d2a-10aa-411e-b55f-513e1f363487.png)

![image](https://user-images.githubusercontent.com/85209746/160240096-663cb2df-9126-4c0a-af2e-4e4874eea922.png)

**Comment by ChesterWOV-DPY** [comment, marked as answer]
You made a typo in the `elif` clause. You said `elif essage.content.startswith` while you should be typing `elif message.content.startswith`.

**Reply by naza1144** [reply]
It works （＾∀＾●）ﾉｼ
